eo-maven-plugin/src/main/java/org/eolang/maven/hash/CommitHash.java

```diff
@@ -23,12 +23,15 @@
  */
 package org.eolang.maven.hash;
 
+import org.cactoos.Scalar;
+
 /**
  * Hash of tag.
  *
  * @since 0.28.11
  */
-public interface CommitHash {
+@FunctionalInterface
+public interface CommitHash extends Scalar<String> {
```

Comment:
@maxonfjvipon I inhereted it firstly because it has the same method so it really can be inherited. Also it allowed to write                     
```java
                    Stream.generate(
                        () -> new ChRemote("0.23.19")
                    ).limit(threads).collect(Collectors.toList())
```
and this is cool.